test/fail_compilation/retscope2.d

```diff
@@ -162,3 +162,67 @@ void foo800()
     scope Exception e;
     throw e;
 }
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Git's text-diff does merge changes to the same file without conflict if there is at least one common line between the changes, so adding newlines helps, but adding more than a couple of newlines seems excessive.